client/src/components/UMDAnnotation.vue

```diff
@@ -88,8 +88,16 @@ export default defineComponent({
         'Finalizing Negotiating/Deal',
         'Refusing a Request',
       ];
+      const normVariants = root
+        .map((item) => [`${item} (adhered)`, `${item} (violated)`])
+        .reduce((acc, x) => acc.concat(x));
+
+      return [
+        'None',
+        ...normVariants,
+      ];
     });
-    const normsObject: Ref<Record<string, 'adhered' |'violate' | 'noann' | 'EMPTY_NA'>> = ref({});
+    const normsObject: Ref<Record<string, 'adhered' |'violated' | 'noann' | 'adhered_violated'>> = ref({});
```

Comment:
We can get rid of `'noann'` from this type.

Reply:
Hm or maybe not. I see you're using `noann` as a sentinel to indicate no emotions selected. It might be better to use a separate value for that (`null`?) since the term "noann" had/has some kind of meaning to NIST.

Reply:
Looking at the documentation again the proper value for 'none' is 'EMPTY_NA' so I'm going to change it to that and make sure the back-end updates any existing 'noann' to the proper 'EMPTY_NA' output.